StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement.swift

```diff
@@ -178,6 +179,14 @@ public final class EmbeddedPaymentElement {
             configuration: configuration,
             loadResult: loadResult
         )
+        self.containerView = EmbeddedPaymentElementContainerView(
+            embeddedPaymentMethodsView: embeddedPaymentMethodsView
+        )
+
+        self.containerView.updateSuperviewHeight = { [weak self] in
```

Comment:
Any reason to not just remove `containerView` and just return it directly on `view`? Guessing it's to avoid casting between UIView and the container view?

Reply:
hm can you elaborate, not sure I understand.